process/smartContract/builtInFunctions/esdtBurn.go

```diff
@@ -100,6 +100,7 @@ func (e *esdtBurn) ProcessBuiltinFunction(
 			core.BuiltInFunctionESDTBurn,
 			vmInput.Arguments,
 			vmInput.RecipientAddr,
+			vmInput.GasLocked,
```

Comment:
Suggested minor refactoring:
* Rename to `addOutputTransferToVMOutput`
* Pass the entire `vmInput`, and change the function to extract the required info

Reply:
can't do that as in case of esdt transfer and smart contract call - there is another arguments list